tests/unittests/suchvim/user-plugin.vader

```diff
@@ -0,0 +1,44 @@
+Before (setUp):
+  source ~/.SUCH-Vim/suchvim/user-plugins.vim
+
+
+Execute (Given user plugin When add plugin Then plugin is added):
+  let plugin_to_add = "ValidGithubUserName/ValidPluginName"
+
+  let actual = SUCHVim_addPlugin(plugin_to_add)
+
+  let expected = "Plug '".plugin_to_add."'"
+  Assert expected == actual[0]
+
+
+Execute (Given users plugins When add plugin Then plugins is added):
```

Comment:
Your tests should be your documentation (sort of). This test does not really add more valuable information compared to the previous. I think you could remove it?